docs/docs/configuration/configuration-sources.md

```
@@ -18,6 +18,20 @@
+## Premium config sources
+
+| Type                                                 | Watch mode    | Import                  | Description                                                                                                                        |
+| ---------------------------------------------------- | ------------- | ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------- |
+| [IORedis](/plugins/premium/config-source/ioredis.md) | Yes           | `@tsed/config/ioredis`  | Load configuration from a Redis database using ioredis. Supports watch mode to reload the configuration when the database changes. |
+| [Mongo](/plugins/premium/config-source/mongo.md)     | Yes           | `@tsed/config/mongo`    | Load configuration from a MongoDB collection. Supports watch mode to reload the configuration when the collection changes.         |
+| [Vault](/plugins/premium/config-source/vault.md)     | Yes - pulling | `@tsed/config/vault`    | Load configuration from a HashiCorp Vault. Supports watch mode to reload the configuration when the vault changes.                 |
+| Postgres - WIP                                       | Yes - pulling | `@tsed/config/postgres` | Load configuration from a Postgres database. Supports watch mode to reload the configuration when the database changes.            |
+
+::: warning Wants more?
+Ask us for a custom config source! We can create a custom config source for you, tailored to your needs.
+Contact us at contact-tsed@gmail.com or via [Slack](https://slack.tsed.io/).
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)